### ISSUE TITLE
Implement a query to check ENTERPRISE/OEM license validity

### ISSUE THREAD

**Issue by gitbuda** [memgraph/memgraph#1058]

**Comment by katarinasupe** [comment]
+1 from the community ([source](https://discord.com/channels/842007348272169002/1210098712085401670/1210137341751074956))

**Comment by DavIvek** [comment]
## Triage Result: Implemented

The query to check license validity has been added.

**Implementation commit:** `318028c6c` — "Add SHOW LICENSE INFO query" (#2555)

You can now run `SHOW LICENSE INFO` to get license details. The grammar rule `licenseInfo : LICENSE INFO` is wired through `systemInfoQuery`.

**Next steps:** Could you verify this meets your needs on the latest Memgraph release?

---
*Automated triage by Memgraph Issue Triage Agent*

**Comment by gitbuda** [comment]
Yep, this is it, we forgot to link/close this issue... -> 💪 for the agent!
